client/src/components/Walkthrough/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement, useState, useEffect } from 'react';
+import { Message } from 'semantic-ui-react';
+
+import { addVisitedPage } from '../../api/user';
+import { useAuth } from '../../contexts';
+
+import './styles.scss';
+
+const Walkthrough = (props: {
+  page: string;
+  content: string;
+}): ReactElement => {
```

Comment:
Would typing this function with `React.FC` and giving it a type parameter of the prop types be a simpler way of typing this?